### PR TITLE
Syndicate cyborgs are no longer able to be flashed

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Syndicate cyborgs can no longer be flashed and added a message for when you fail to flash a borg (there previously wasn't one)

## Why It's Good For The Game
Syndicate cyborgs can get one shot by almost anyone on the station, all you have to do is stunlock them with flashes and beat them to death. They have this instant stun combat because normalborgs can get instant stuns to fight back against flashes, syndicate cyborgs cant. Combat cyborgs (at least the assault cyborg) should be resistant to flashes so they can actually fight on the front line without being flashed and pummeled with no counters. They would still have weaknesses to ion shots (from the ion rifle, HOS gun, or any ion carbines made with RND, and EMPs which can be made pretty easily with chemistry).

## Testing

Couldn't flash any syndicate cyborg and got the new special message instead

## Changelog
:cl:
tweak: Nuclear Operative cyborgs can no longer be flashed, use an ion or EMP instead.
/:cl:

### CODE REVIEW

**Comment by Coolrune206** [comment]
Nukie borgs are intended to be support units, whether it be stealth sabotage, direct medical, or fire support. If a Nukie Cyborg gets flashed, that means that the Cyborg is either overextended away from the team it is meant to support, or the Nukie team has failed to protect their Cyborg support from a melee-ranged attacker. I understand it can be frustrating to experience being flashed and killed as a Nukie borg, but in my honest opinion them being vulnerable to flashes seems fine to me for the above stated reason.

File: code/modules/mob/living/silicon/robot/syndicate.dm [hunk 1]
```diff
@@ -137,3 +138,6 @@
 	if(cham_proj)
 		cham_proj.disrupt(src)
 	..()
+
+/mob/living/silicon/robot/syndicate/death(gibbed)
+	. = ..()
```
Comment (Contrabang) [inline code/modules/mob/living/silicon/robot/syndicate.dm:143]: I forgot I committed this bit. I was looking at tradeoffs

Todo: remove

**Comment by matttheficus** [comment]
I dislike the idea of permanent flash protection. Perhaps a "recharging" protection (say, it blocks a flash or two before needing to fall back and let it come back online like the Shielded Hardsuit) so that they are required to still act as Support units rather than frontline juggernauts with no slowdown.

**Comment by hal9000PR** [comment]
considering the ion oneshots them from a range with an AOE I am fine with this.

**Comment by Denthamos** [comment]
I am more on the side of flashes should still effect borgs, but not be an instant stun.  Let a flash do what it does, flash and blind the borg just like organics.  If it gets too many flashes done to it, the optical sensor can become damaged and need fixing/replacement from robotics.  I am saying this for **all** cyborgs, not just syndie borgs.

**Comment by matttheficus** [comment]
After thinking on it and seeing what Denth said, I think I'd actually agree with that as well. Making flashes eventually burn out optical sensors but not stunning them anymore might actually work.

**Comment by SpringSkipper** [comment]
> Nukie borgs are intended to be support units, whether it be stealth sabotage, direct medical, or fire support. If a Nukie Cyborg gets flashed, that means that the Cyborg is either overextended away from the team it is meant to support, or the Nukie team has failed to protect their Cyborg support from a melee-ranged attacker. I understand it can be frustrating to experience being flashed and killed as a Nukie borg, but in my honest opinion them being vulnerable to flashes seems fine to me for the above stated reason.

Assault borg should not be grouped in like that.

I get the team-based antag argument, but we've moved away from stun combat which this is a clear holdover of. Normal borgs are flash-vulnerable because - at least in theory - they need to be possible to detain and de-subvert. Nukies are a case of kill-or-be-killed. They don't need to be able to be easily detained (in fact they have bomb implants to ensure the opposite!) and their borgs should be no different. Syndie borgs aren't invulnerable without the aid of a freebie stun option; they're perfectly able to just be outnumbered and killed in the traditional manner if left out alone.

A 71 TC assault borg should simply not be instastunned with a laser pointer. It just shouldn't. This isn't feelbads or I-lost-plz-nerf. This isn't a strategically-placed vulnerability on what is otherwise an unstoppable juggernaut with no need of support. This isn't an argument that I think needs to even be made, but okay:  For 71 TC (borg + vtec) you can hire a whole extra nukie (20 TC) and outfit them with adrenals, an m90, extra grenades, emag, medkit, and four extra clips of ammo plus any extras left over on the nukie shuttle. There's simply no comparison.

Flash vulnerability on other borgs is much less absurd compared to the assault, because they're actually meant to be supports and not a spearhead. I believe the syndie mediborg even gets the ~bullshit~ highly robust combat defib, in addition to some extremely strong support options that an equivalent support-oriented nukie simply can't get access to. Meanwhile the saboborg's main contribution is their low price for the power of silicon access. It still feels like shit that if you try to ambush any random crew member you've fooled they might just have a flash on them, click you once, and win even though you got three hits on them first. But hey, balance will never be a pinpoint for this game.

**Comment by hal9000PR** [comment]
well I got outvoted on this one.
2 objections, only one in favour. moving to closed

**Pull request closed without merging**